test/mocha/20-key-operations.js

```diff
@@ -115,6 +136,27 @@ describe('bedrock-kms', () => {
         const {signatureValue} = result;
         signatureValue.should.be.a('string');
       });
+      it('signs a string using Ed25519VerificationKey2020', async () => {
+        const {id: keyId} = await helpers.generateKey(
+          {mockData, type: 'Ed25519VerificationKey2020'});
+        const operation = clone(mockData.operations.sign);
+        operation.invocationTarget = keyId;
+        operation.verifyData = uuid();
+        let result;
+        let error;
+        try {
+          result = await brKms.runOperation({operation});
+        } catch(e) {
+          error = e;
+        }
+        assertNoError(error);
+        should.exist(result);
+        result.should.be.an('object');
+        Object.keys(result).should.have.same.members(['signatureValue']);
```

Comment:
this can just be `result.should.have.keys(['signatureValue']);`  Here and on L52.

Reply:
addressed here d929fbf